### DISCUSSION TITLE
Can I use the 200A CTs to measure individual circuits?

### DISCUSSION THREAD

**Discussion by gminco** [emporia-vue-local/esphome#272] [Q&A] [answered]
I have solar installed, and I can connect locally to my Enphase gateway to get the consumption readings (I created sensors using rest while connected locally).

Can I repurpose the 200A CTs to read from individual circuits?
(I bought an 8 clamp vue 2, and I'm hoping to read 10 circuits).

Thanks!

**Comment by flaviut** [comment]
Yup, [someone reported success using a 3.5 to 2.5mm adapter](https://github.com/emporia-vue-local/esphome/discussions/36#discussioncomment-3724317). [You'd need to use the filters to scale the value.](https://esphome.io/components/sensor/index.html#calibrate-linear)

**Reply by gminco** [reply]
Thanks, @flaviut!
But, couldn't somehow use the 3.5 jacks from the top and just configure them as ct_clamps sensors?

**Reply by flaviut** [reply, marked as answer]
Sure, in that case you just need to change the id/names.

**Reply by fctruter** [reply]
Sorry for my ignorance, but where in the YAML file should we change the id/names? Also, how do I then remove the Total Power & Total Daily Energy sensors and add a normal Daily Energy sensor for the two 200A CT's?

**Reply by flaviut** [reply]
Change the ID here:

```
        power:
          id: phase_a_power
          filters: [*pos]
```

and the name here:

```
  - { platform: copy, name: "Phase A Power", source_id: phase_a_power, filters: *throttle_avg }
```

keep in mind you must keep these two parts of the config in sync.

To delete a sensor, you just need to delete that element from the sensor list. This guide may be helpful to understand how YAML works: https://circleci.com/blog/what-is-yaml-a-beginner-s-guide/. I'm not sure exactly what you are looking to do, but you can use ["Template Sensors"](https://esphome.io/components/sensor/template.html) to do any kind of arithmetic you'd like. Just look at the existing code & modify it to suit your goal.
